ros2bag/ros2bag/verb/play.py

```diff
@@ -47,7 +51,7 @@ def check_positive_float(self, value):
             raise ArgumentTypeError('%s is not of the valid type (float)' % value)
 
     def main(self, *, args):  # noqa: D102
-        bag_file = args.bag_file
+        bag_file = args.input
```

Comment:
Don't forget this too

Reply:
Haha good catch. I luckily deleted it when I merged from master.